docs/timeseries.md

```diff
@@ -1,122 +1,275 @@
+#### min(column)
 
-### Mutation
+Returns the minimum value found in a given column. If a column is not supplied it assumes the column to be "value". We often use this to generate axis scales when charting the TimeSeries.
 
-Although the TimeSeries is immutable itself, these operations will return a new modified TimeSeries:
+#### mean(column)
 
- * `slice(begin, end)` - will return a new TimeSeries with reference just to the Events that were left after the slice. The result represents a portion of this TimeSeries from begin up to but not including end.
+Returns the mean of the Timeseries values for the given column. If a column is not supplied it assumes the column to be "value". This is the same as taking the avg().
+
+#### medium(column)
+
+Returns the medium of the Timeseries values for the given column. If a column is not supplied it assumes the column to be "value".
+
+#### stddev(column)
+
+Returns the standard deviation of the Timeseries values for the given column. If a column is not supplied it assumes the column to be "value".
+
+---
+
+### Mutation API
+
+Although the TimeSeries is immutable itself, these operations will return a new modified TimeSeries. We expect to add several more functions here over time.
+
+#### slice(begin, end)
+
+Returns a new TimeSeries with reference just to the Events that were left after the slice. The result represents a portion of this TimeSeries from begin up to but not including end.
 
 For example:
 
-    const begin = series.bisect(t1);
-    const end = series.bisect(t2);
-    const cropped = series.slice(begin, end);  // Returns a cropped series
+```javascript
+const begin = series.bisect(t1);
+const end = series.bisect(t2);
+const cropped = series.slice(begin, end);  // Returns a cropped series
+```
+
+#### TimeSeries.merge(info, [series1, series2, ...]) [Static]
+
+Returns a new TimeSeries that merges a list of TimeSeries together. The common uses for this are:
+
+ * Join a list of `TimeSeries` together that have different columns.
+ * Append together a list of `TimeSeries` with the same columns, but different times.
+
+The `info` passed into the `merge()` function is an Object which is used to create the new `TimeSeries`. It contains a name, index (optional), and meta data.
+
+There are some rules surrounding the use of `merge()`.
+
+If we consider each row of each `TimeSeries` with the same `Date` (or `Index`, or `TimeRange`), then we have a list of `Events` (or `IndexedEvents`, or `TimeRangeEvents`). This list needs to be reduced. To do this the events themselves are merged using `Event.merge()`. This operation will not attempt to reduce values which have the same column, for the same time, so for each of these lists of events there should be no shared columns. In other words, you can merge a `TimeSeries` with columns "a" and "b" with a `TimeSeries` with a column "c", but not with a `TimeSeries` with a column "a" (if the two `TimeSeries` overlap their times).
```

Comment:
at the end of this line, it's probably clearer if you say "but not a `TimeSeries` with a column "a" or "b" (if the two `TimeSeries` overlap their times).